<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scoremanage</class>
 <widget class="QWidget" name="scoremanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>GRADE MANEGEMENT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SNO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OBJECT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_objectlineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_IDlineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_teacherlineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_scorelineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="score_querypushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="score_upadtepushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="score_backpushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>SNO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CLASS</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SCORE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="courseid_lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FIND BY COURSE ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_all_scoreButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SHOW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="asc_Button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ASC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="desc_Button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DESC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ORDER BY SCORE:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
